app/javascript/pages/getAssistance/counselor-filter.scss

```diff
@@ -19,7 +19,7 @@ select {
     }
     select {
         border-color: var(--bloom-color-gray-500);
-        @apply text-gray-700;
+        color: #555555;
```

Comment:
We should use the built-in style values instead of hardcoded ones. In this case we just need to change it to `@apply text-gray-750`

Reply:
For component-specific stylesheets, we've been using `@apply` instead of `color` for text. I'm not sure why we do it one way or the other, but we should keep it consistent unless there is a good reason to change.

Reply:
thanks!